prusti-tests/tests_old/parse/ui/attribute_macros.stderr

```diff
@@ -13,7 +13,7 @@ error: malformed specification (expected single argument)
 error: malformed specification (expected token)
   --> $DIR/attribute_macros.rs:15:5
    |
-15 |     #[invariant["true"]]
+15 |     #[invariant["true)]]
```

Comment:
Seems like a rogue regex replacement. I see this is in `tests_old`, but still. This file also has assertions in strings still.

Reply:
True. I'll revert the change, but I'm not updating the annotations to the new syntax because I think it's easier to regenerate them by running Prusti and manually checking the output.